packages/api/src/platforms/vtex/utils/facets.ts

```diff
@@ -1,16 +1,32 @@
+import ChannelMarshal from './channel'
+
 export interface SelectedFacet {
   key: string
   value: string
 }
 
 /**
  * Transform facets from the store to VTEX platform facets.
- * For instance, the channel in Store becomes trade-policy in VTEX's realm
+ * For instance, the channel in Store becomes trade-policy and regionId in VTEX's realm
  * */
 export const transformSelectedFacet = ({ key, value }: SelectedFacet) => {
   switch (key) {
-    case 'channel':
-      return { key: 'trade-policy', value }
+    case 'channel': {
+      const channel = ChannelMarshal.parse(value)
+
+      const result = []
+
+      if (channel.salesChannel) {
```

Comment:
there's always a salesChannel, so no need for this `if` in here

Reply:
it's always passed because we always set it on store.config. Also, the store always have sc=1 available

Reply:
Yeah, you're right, but this function is used inside the resolver ( that receives facets and isn't required to have a facet called channel with salesChannel. Does it make sense?
https://github.com/vtex/faststore/blob/447a842a05e9e2e18b6a0adc11797ec410278036/packages/api/src/platforms/vtex/resolvers/query.ts#L47
https://github.com/vtex/faststore/blob/447a842a05e9e2e18b6a0adc11797ec410278036/packages/api/src/platforms/vtex/resolvers/query.ts#L29

Reply:
but the `if` is already on the switch-case statement, isn't it?

Reply:
Like, if the channel was passed, we know for sure we have a salesChannel. Does it make sense?